code/datums/components/breeding.dm

```diff
@@ -51,7 +51,7 @@
 	if(source.combat_mode)
 		return
 
-	if(source.client || target.client)
+	if(source.client || (ismob(target) && target.client))
```

Comment:
Should this have an istype() check before?

You could still pass a non mob object into this proc as an arg.